### PR TITLE
Update release-note with Graphql Tool changes

### PR DESCRIPTION
**Pull request by udda1996**
## Purpose
$title

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 1]
```diff
@@ -80,12 +80,19 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to connector configuration
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:84]:
```suggestion
- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 2]
```diff
@@ -80,12 +80,19 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
...
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
+- Added IDL client generation support using non-Ballerina Interface Definition Language
+
 ### Improvements
 
 #### Ballerina shell
 
 #### Ballerina update Tool
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:91]:
```suggestion
#### Ballerina Update Tool
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 3]
```diff
@@ -80,12 +80,19 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
+- Added IDL client generation support using non-Ballerina Interface Definition Language
+
 ### Improvements
 
 #### Ballerina shell
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:89]:
```suggestion
#### Ballerina Shell
```

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 4]
```diff
@@ -80,11 +80,18 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
+- Added IDL client generation support using non-Ballerina Interface Definition Language
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:85]:
```suggestion
- Added support to generate graphQL client definitions using graphQL IDL(i.e. config) files
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md [hunk 5]
```diff
@@ -80,11 +80,18 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:84]: IMO we can consider this as an improvement and say `Improved graphQL client config generation to support runtime configurability`

Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:84]: Should be: `Improved the GraphQL client config generation to support runtime configurability`.

File: release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md (hunk 4 above)
Comment (praneesha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md:85]: Should be: `Added the support to generate GraphQL client definitions using GraphQL IDL(i.e., config) files`

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Review by praneesha — APPROVED** [review]

**Pull request merged**